docker/ckb/entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+if ! [ -f ckb.toml ]; then
+  /bin/ckb init --chain "$CKB_CHAIN" --ba-arg "$BA_ARG" --ba-code-hash "$BA_CODE_HASH" --ba-hash-type "$BA_HASH_TYPE" --ba-message "$BA_MESSAGE"
+fi
+
+exec /bin/ckb run &
```

Comment:
Does the logs of `ckb run` redirect to docker log in this way?

Reply:
fixed